.gitignore

```diff
@@ -118,6 +118,8 @@ tests/Resources/img/c/*
 !tests/Resources/img/c/.gitkeep
 tests/Resources/img/tmp/*
 tests/Resources/img/tmp/.gitkeep
+tests/Resources/img/m/*
+!tests/Resources/img/m/.gitkeep
```

Comment:
Images are automatically created in `tests/Resources/img` maybe ignore the whole directory instead

Reply:
Well that's what the `*` rule does, it ignores the whole content, but we still have a `.gitkeep` file in the folder that is on the repository so it shouldn't be ignored (even if it's already added so it doesn't change much) All our empty folders are usually handled this way in gitignore